### ISSUE TITLE
Index USB Connection Not Recognized

### ISSUE THREAD

**Issue by TheVeridian** [aaronanderson/LinuxVMWindowsSteamVR#2]
Hello, creating this to hopefully seek advice on an issue regarding the HMD connecting to SteamVR:

My VM is setup and ready, all drivers and passthroughs working, however SteamVR isn't picking up the USB for the Index, but it works on other physical hardware. Here is the system report from SteamVR, showing what it's picking up for USB devices on the PCI-e card:

```
Renesas Electronics USB 3.0 Host Controller

Manufacturer: Renesas Electronics, Provider: Renesas Electronics, 8/21/2012, 3.0.23.0

        Root Hub

            [Port1]

            [Port2]

            [Port3]: Renesas Electronics USB 3.0 Hub (VID:0424 PID:5744)

                [Port1]: USB Composite Device (VID:28DE PID:2400), Etron Technology, Inc., 3D Camera, 00000001

                [Port2]

                [Port3]

                [Port4]

            [Port4]

            [Port5]: USB Input Device (VID:28DE PID:2101), Valve Software, Watchman Dongle, 22CEDF54BC

            [Port6]: USB Input Device (VID:28DE PID:2101), Valve Software, Watchman Dongle, 4235D4424C

            [Port7]

            [Port8]: USB Input Device (VID:28DE PID:2101), Valve Software, Watchman Dongle, 85B732B972
```

Is there something I missed when setting this up? Thx a ton for your guide.

**Comment by aaronanderson** [comment]
That is strange, SteamVR shouldn't be aware of the virtualization as it is done at the OS level. 

In the guest Windows device manager are there are any driver warnings? 

Here are some Valve provided support instructions from this thread: https://steamcommunity.com/app/1059530/discussions/0/1640915206486950004

1. Restart your computer and plug your Index into a USB 2.0 port. 
2. Power cycle your Index; Remove the AC connection for 30 seconds.
3. Reset your SteamVR USB devices from Menu (3 lines) > Developer > Developer Settings > In the 'Reset' subsection, click 'Remove all SteamVR USB Devices'.
4. Disable Windows Enhanced Power Management for SteamVR devices from the same menu by clicking on "Disable Power Management."
5. Shut down SteamVR.
6. Restart your computer
7. Reconnect your VR equipment to USB 3.0+ ports, start SteamVR and retest the issue.

I would try #1 and #2 on your host, if you even have a USB 2.0 port anymore, and then the rest in the guest OS.

**Comment by TheVeridian** [comment]
Thank you so much for your reply, here's some quick details about the VM setup:
There are no driver warnings in device manager, and via host OS (Arch Linux) I have QEMU spoofing the board ID and all of the manufacturer info to my actual physical hardware so that Windows doesn't detect that it's virtualized and EAC works.

As for the steps you provided, how would I pass the index into the VM via a USB 2.0 port? Passing through my front panel I/O? The PCI-e card I have only exclusively has 4x 3.0 parts. In the first comment you made, should it not be detected as a reneseas host controller? I did manually install the driver for it via their official website..

I have attempted to do #3 and #4, but it looks like for some reason through the PCI-e card it's just not picking up the entire index, only the cameras. Based on that Valve thread link it looks like I should be seeing these too:
```
[Port1]: USB Composite Device (VID:28DE PID:2300), Valve, Index HMD, LHR-FFE48FEA
[Port2]: USB Composite Device (VID:28DE PID:2102), Valve Corporation, Valve VR Radio, 8FE68CE063-RYB
[Port3]: USB Composite Device (VID:28DE PID:2102), Valve Corporation, Valve VR Radio & HMD Mic, 48EB3BCE86-LYM
```

Thank you so much for your help, again
